applications/API/urls.py

```
@@ -0,0 +1,20 @@
+from applications.API import views
+
+router = DefaultRouter()
+router.register(r"insults", views.InsultViewSet, basename="insult")
+router.register(r"categories", views.CategoryViewSet, basename="category")
+
+urlpatterns = [
+    # API documentation/schema endpoints
+    path("schema/", SpectacularAPIView.as_view(), name="schema"),
+    path("swagger/", SpectacularSwaggerView.as_view(), name="swagger"),
+    path("redoc/", SpectacularRedocView.as_view(), name="redoc"),
+] + router.urls
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

(RUF005)